spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -164,14 +164,13 @@ def factory(attrs)
         # @param application (Spaceship::Tunes::Application) The app this version is for
         # @param app_id (String) The unique Apple ID of this app
         # @param is_live (Boolean)
-        def find(application, app_id, is_live)
+        def find(application, app_id, is_live, platform: nil)
           # we only support applications
-          platform = Spaceship::Tunes::AppVersionCommon.find_platform(application.raw_data['versionSets'])
-          raise "We do not support BUNDLE types right now" if platform['type'] == 'BUNDLE'
+          raise "We do not support BUNDLE types right now" if application.type == 'BUNDLE'
```

Comment:
@mpirri Not sure what the solution would be here. Won't application.type always have a value?